### DISCUSSION TITLE
How to set a certain includePath when there are multiple header with same filename ?

### DISCUSSION THREAD

**Discussion by Mactarvish** [microsoft/vscode-cpptools#9942] [Q&A] [answered]
Hi,
there are two set of Qt in my mac:
`/Users/titan/Qt/6.4.0/macos/lib/` (Qt 6.4)
`/Users/titan/opt/anaconda3/include/qt/` (Qt 5.13, the same header filename with Qt6.4 while the contents are different)

I'm using the first one (Qt 6.4), but I found that the cpptools always opens the header file of the second one (Qt 5.13) when I ctrl-click `#include <qtxxx>` , any way to set the correct searching path? Thanks.

By the way, I've tried add `"/Users/titan/Qt/6.4.0/macos/lib/**" ` to `c_cpp_properties.json` manually, but it doesn't fix my problem.

**Comment by sean-mcmanus** [comment, marked as answer]
Add "/Users/titan/opt/anaconda3/include/qt" to the C_Cpp.files.exclude setting or set the browse.path such that that folder isn't included or put /Users/titan/Qt/6.4.0/macos/lib (non-recursive) before the other path (you may need to add other non-recursive paths too).
